db/migrate/20170127013809_add_slug_to_eventos.rb

```diff
@@ -0,0 +1,5 @@
+class AddSlugToEventos < ActiveRecord::Migration
+  def change
+    add_column :eventos, :slug, :string
+  end
```

Comment:
Acho que o slug seria só pra notícia mesmo, pra ficar no URL. No caso do evento podemos usar um id_amigavel (já que vai ter um id numérico automático).